xbmc/media/MediaType.cpp

```diff
@@ -75,6 +75,14 @@ MediaType MediaTypes::ToPlural(const MediaType &mediaType)
   return mediaTypeIt->second.plural;
 }
 
+bool MediaTypes::IsCollection(const MediaType &mediaType)
```

Comment:
Since we already have `MediaTypeVideoCollection` I'd suggest to drop this and simply use what's there.